Makefile

```diff
@@ -0,0 +1,41 @@
+.PHONY: build test clean prepare update docker
+
+GO = CGO_ENABLED=0 GO111MODULE=on go
+
+MICROSERVICES=cmd/device-gpio-go
+
+.PHONY: $(MICROSERVICES)
+
+DOCKERS=docker_device_gpio_go
+.PHONY: $(DOCKERS)
+
+VERSION=$(shell cat ./VERSION 2>/dev/null || echo 1.0.0)
+GIT_SHA=$(shell git rev-parse HEAD)
+GOFLAGS=-ldflags "-X github.com/edgexfoundry/device-gpio-go.Version=$(VERSION)"
+
+build: $(MICROSERVICES)
+	$(GO) build ./...
+
+cmd/device-gpio-go:
+	$(GO) build $(GOFLAGS) -o $@ ./cmd
+
+test:
+	$(GO) test ./... -coverprofile=coverage.out
+	$(GO) vet ./...
+	gofmt -l .
+	[ "`gofmt -l .`" = "" ]
+	./bin/test-attribution-txt.sh
+	./bin/test-go-mod-tidy.sh
+
+
+clean:
+	rm -f $(MICROSERVICES)
+
+docker: $(DOCKERS)
+
+docker_device_gpio_go:
+	docker build \
+		--label "git_sha=$(GIT_SHA)" \
```

Comment:
Add these so those behind a proxy can build docker image.
```
        --build-arg http_proxy \
        --build-arg https_proxy \
```
If the proxy is set it uses it, otherwise it is ignored.

Reply:
+1 this is fixed